pysma/definitions.py

```diff
@@ -61,14 +61,20 @@
 pv_power_a = Sensor("6380_40251E00_0", "pv_power_a", unit="W")
 #: Current power generated by the solar panels (B side)
 pv_power_b = Sensor("6380_40251E00_1", "pv_power_b", unit="W")
+#: Current power generated by the solar panels (C side)
+pv_power_c = Sensor("6380_40251E00_2", "pv_power_c", unit="W")
```

Comment:
```suggestion
pv_power_c = Sensor("6380_40251E00_2", "pv_power_c", unit="W", enabled=False)
```

Reply:
That's fine, although I don't really see this any different than other sensors which are enabled by default and not all inverters have. For example, my inverter doesn't support 11 of the existing sensors that are enabled by default and which I disable manually.

I suspect the current set is probably based around what your own inverter supports. And that's fine, but I'm not sure that translates to everyone any better. I would argue it's easier to have them all enabled by default and let people turn off the ones that don't apply to them than it is to have sensors disabled and try and figure out which ones are needed. I also understand you own this integration so you are free to choose the defaults you want. :-)

Thanks!

Reply:
I've by default disabled quite a few sensors in #78 as requested by the HA team.

Reply:
I've updated the PR with the sensors I added disabled by default.

Even with #78, there are many such as everything having to do with L3, or most of the yield sensors, which don't apply for my particular installation and/or model. But, I am not trying to have an argument, just mention that as with all defaults, there is no such thing as good defaults. :-) Thanks again!